install_files/ansible-base/roles/restrict-direct-access/tasks/fetch_tor_config.yml

```diff
@@ -26,6 +28,42 @@
   # Local action, so we don't want elevated privileges
   become: no
   with_items: "{{ tor_hidden_service_hostname_lookup.results }}"
+  when: "v2_onion_services|default(True)"
+  tags:
+    - tor
+    - admin
+
+
+- name: Wait for all Tor v3 onion services hostname files.
+  wait_for:
+    state: present
+    path: "{{ tor_hidden_services_parent_dir }}/{{ item.service }}/hostname"
+    delay: 5
+  with_items: "{{ tor_instances_v3 }}"
+  when: "v3_onion_services|default(False)"
+  tags:
+    - tor
+
+- name: Collect Tor v3 onion service hostnames.
+  command: cat /var/lib/tor/services/{{ item.service }}/hostname
+  register: tor_hidden_service_hostnamev3_lookup
+  # Read-only task, so don't report changed.
+  changed_when: false
+  with_items: "{{ tor_instances_v3 }}"
+  when: "v3_onion_services|default(False)"
```

Comment:
As above, rather than repeating `|default(False)`, set `v3_onion_services: False` in `install_files/ansible-base/group_vars/all/securedrop`. Then the `sdconfig` command can update that value to True, in line with the recommendations in #4628.

Reply:
I will do this.